src/Workspaces/Core/Portable/Classification/ClassifiedSpan.cs

```diff
@@ -8,22 +8,16 @@
 
 namespace Microsoft.CodeAnalysis.Classification
 {
-    public readonly struct ClassifiedSpan : IEquatable<ClassifiedSpan>
+    public readonly struct ClassifiedSpan(TextSpan textSpan, string classificationType) : IEquatable<ClassifiedSpan>
```

Comment:
>ClassifiedSpan [](http://example.com/codeflow?start=27&length=14)

Wouldn't a record be better?

Reply:
Let me see about updating the feature to support that afterwards!